test/cppsuite/tests/run.cpp

```diff
@@ -88,6 +88,7 @@ print_help()
     std::cout << std::endl;
     std::cout << "SYNOPSIS" << std::endl;
     std::cout << "\trun [OPTIONS]" << std::endl;
+    std::cout << "\trun -H [HOME]" << std::endl;
```

Comment:
Can we A->Z please

Reply:
[eb6d460](https://github.com/wiredtiger/wiredtiger/pull/8484/commits/eb6d460cf9f91e632783d4cbe5537064096d2264)